### ISSUE TITLE
Migrate Play telemetry samples to this repo

### ISSUE THREAD

**Issue by ignasi35** [lightbend/telemetry-samples#15]
Move https://github.com/playframework/play-samples/blob/2.8.x/play-scala-telemetry-example and 
https://github.com/playframework/play-samples/blob/2.8.x/play-java-telemetry-example to this repository.

**Comment by ignasi35** [comment]
Consider moving the `2.7.x` version of those samples too(?)

**Comment by marcospereira** [comment]
> Consider moving the 2.7.x version of those samples too(?)

I don't think so. Let's keep this focused on the latest versions only (so we won't need to maintain samples for Lagom 1.5, Akka 2.5, etc).
